### ISSUE TITLE
The onRender method is not called on the 31st to 100th items in the list view.

### ISSUE THREAD

**Issue by fujii115** [SharePoint/sp-dev-docs#6493]
## Category

- [x] Bug

## Describe the bug

When displaying the FieldCustomizer column in the list view, onRender method may not be called for the 31st to 100th items.
The onReander method is called for the 1st to 30th cases and the 101st and subsequent cases.

This bug occurs frequently when the page is displayed for the first time.
In most cases, reloading the page will call onRender, but reloading may not resolve it.

If you change the number of items displayed in the list view settings from 30 (default) to 100, the problem will no longer occur.

Has anyone encountered this bug?

## Steps to reproduce

1. Open a list with more than 101 items
2. Display a list view with a FieldCustomizer column
3. Scroll to the 1st to 100th items

## Expected behavior

The onRender method is called without omission for all items.

## Environment details (*development & target environment*)

> **DELETE THIS LINE BEFORE SUBMITTING** - Complete the following (*if ignored you'll be prompted for it before we can address your issue... save the time and provide it in your initial submission*).

- **Your Developer Environment**: Windows 10 
- **Target Environment**: SharePoint Online
- **Framework**: Node.js
- **Browser(s)**:Chrome , FireFox , IE v11 , Edge ,Chromium Edge 
- **Tooling**: SPFx v1.10.0

**Comment by AJIXuMuK** [comment]
Hi @fujii115 - sorry for the long delay.
The issue has been fixed a while ago.
Please feel free to open a new issue if the problem still exists on your environment.
